scipt.js

```diff
@@ -0,0 +1,6 @@
+const hamburger = document.querySelector('.menu-bars');
+const mobileMenu = document.querySelector('#mobile-nav');
+
+hamburger.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
```

Comment:
- [ ] Nice JS work here on hamburger functionality, however, your project is still lacking the functionality for a closing button as asked in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) 🔽, please add that.
> When the user clicks (or taps) the close (X) button, the mobile menu disappears.